locale/en_GB/LC_MESSAGES/django.po

```diff
@@ -129,48 +129,28 @@ msgstr "Help us improve this service"
 msgid "survey.link.text"
 msgstr "Complete our short survey"
 
-#: ds_judgements_public_ui/templates/includes/judgment_text_download_options.html
-msgid "judgment.downloadoptions.title"
+#: ds_judgements_public_ui/templates/includes/document_text_download_options.html
+msgid "document.downloadoptions.title"
 msgstr "Download options"
 
-#: ds_judgements_public_ui/templates/includes/judgment_text_download_options.html
-msgid "judgment.downloadoptions.pdf.cta"
-msgstr "Download this judgment as a PDF"
+#: ds_judgements_public_ui/templates/includes/document_text_download_options.html
+msgid "document.downloadoptions.pdf.cta"
+msgstr "Download this document as a PDF"
 
-#: ds_judgements_public_ui/templates/includes/judgment_text_download_options.html
-msgid "press_summary.downloadoptions.pdf.cta"
-msgstr "Download this press summary as a PDF"
-
-#: ds_judgements_public_ui/templates/includes/judgment_text_download_options.html
-msgid "judgment.downloadoptions.pdf.description"
-msgstr ""
-"The original format of the judgment as handed down by the court, for "
-"printing and downloading."
-
-#: ds_judgements_public_ui/templates/includes/judgment_text_download_options.html
-msgid "press_summary.downloadoptions.pdf.description"
```

Comment:
here you can see the removal of 2 trnaslations (1 for each of judgment and press summary) to 1 that uses the general 'document' term.